### DISCUSSION TITLE
Confusion about elevation values

### DISCUSSION THREAD

**Discussion by Jinbow-0217** [SynxFlow/SynxFlow#15] [Q&A] [answered]
Hello, may I ask how to deal with the grid elevation value in the **z.dat** file? 
The feeling is not the original elevation value, nor does it seem to be the result of standard normal processing.
Please give me some guidance, thank you.

**Comment by xiaxilin** [comment]
Hi, did you mean the debris flow model? During the processing, the min value was set to 0.

**Reply by Jinbow-0217** [reply]
No, I mean the z.dat file in the **flood simulation model**.
Here's what the file says:
$Element Number
43512
$Element_id  Value
0 11.965
1 13.845
2 15.14
3 15.565
4 12.5
5 11.255

**Comment by xiaxilin** [comment, marked as answer]
Hi, in the flood simulation, the values in z.dat should be the same as the input DEM file.

**Reply by Jinbow-0217** [reply]
Thanks for Dr. Xia's reply, may I ask how the elevation value of the **DEM grid** is set? 
The data should not be the actual elevation value, but relative to a certain observation point?

**Reply by xiaxilin** [reply]
Hi, this can be the actual elevation value, which depends on the digital elevation model that you use.

**Reply by Jinbow-0217** [reply]
Haha, when I look at the specific figures, I think it is some kind of statistical processing. Thanks for your reply, Dr.Xia.
